content/500-platform/10-about.mdx

```diff
@@ -5,13 +5,133 @@ metaDescription: 'Learn about the Platform Console to integrate the Prisma Data
 displayed_sidebar: platformSidebar
 ---
 
-<TopBlock>
+## Overview
 
-Platform Console enables you to integrate the latest Prisma Data Platform products into your application
+Platform Console enables you to integrate the latest Prisma Data Platform (PDP) products into your application
 
 - [Accelerate](/accelerate/what-is-accelerate): Global database cache with scalable connection pooling
 - [Pulse](/pulse/what-is-pulse): A managed Change Data Capture (CDC) service that enables real-time database subscriptions
 
-[Go to Platform Console](https://console.prisma.io)
+Open the [Platform Console](https://console.prisma.io).
 
-</TopBlock>
+## Concepts
+
+The Platform Console workflows are based on four main concepts:
+
+- [**Account**](#account): In order to use PDP products, you need to have a PDP account. An _organization_ will typically create one account to manage all their workspaces, projects and environments.
+- [**Workspaces**](#workspace): An account can have multiple workspaces. A workspace typically represents a _team_ within an _organization_. **Billing happens on a per-workspace basis**, i.e. an _organization_ receives an invoice at the end of the month that captures all cost associated with the projects in a given workspace.
+- [**Project**](#project): A workspace can have multiple projects. A project typically represents the _application_ of a _product_ or _service_ a team is working on.
+- [**Environments**](#environment): A project can have multiple environments. An environment typically maps to a _development stage_, like `dev`, `staging` or `prod`. **API keys are provisioned on the environment-level**.
+
+Here is a visual illustration of how these concepts relate to each other:
+
+![](/img/platform/pdp-concepts.png)
```

Comment:
Could we rather use ACME as the workspace, where there might be a different projects (apps, services, etc)? I think it'd be a bit simpler, and would map better to most of our users.

Reply:
Agree, will update the illustration.

Reply:
how about this?

![image](https://github.com/prisma/docs/assets/4058327/2e5979de-7744-4f0e-a4d5-bc0dd88584d9)

there are still rwp ACME-related workspaces, do you think one of them should be renamed/removed or is it ok like this?

Reply:
I think this is better! I like the differentiation between personal and work workspaces